geoengine/workflow.py

```diff
@@ -496,8 +496,8 @@ def save_as_dataset(
             bbox: QueryRectangle,
             name: str,
             description: str = '',
-            timeout: int = 3600) -> StoredDataset:
-        '''EXPERIMENTAL: Store the workflow result as a layer'''
+            timeout: int = 3600) -> TaskId:
```

Comment:
I guess the library would be nicer if we would return a Task that encapsulates the TaskId.
Then, we could call something like info() or wait_for_result on the task rather than putting the TaskId into ge.tasks.X.
What do you think?

Reply:
5e5dc707d346e16a4188265c652bb632ca0043c6